site/docs/v1/tech/installation-guide/kickstart.adoc

```diff
@@ -279,9 +279,13 @@ In this example we will modify the restricted API key example from above to furt
 
 === Referencing Default Entities
 
-Review the link:https://fusionauth.io/docs/v1/tech/reference/limitations/#default-entities[default entities] documentation for entities with fixed Ids such as the default theme or FusionAuth application. 
+There are some entities in FusionAuth with fixed Ids, such as the default theme or FusionAuth application.
```

Comment:
Using the word `entity` can be confusing as it shares the same namespace as `entities`, `entity types`, and `entity management`

Reply:
fair